packages/lab/stories/carousel/carousel.stories.tsx

```diff
@@ -1,41 +1,196 @@
 import type { Meta, StoryFn } from "@storybook/react";
 
-import { Carousel, CarouselSlide } from "@salt-ds/lab";
+import {
+  Carousel,
+  CarouselControls,
+  CarouselSlide,
+  CarouselSlider,
+} from "@salt-ds/lab";
 import "./carousel.stories.css";
-import { Button, FlexLayout } from "@salt-ds/core";
+import { Button, H2, H3, Link, StackLayout, Text } from "@salt-ds/core";
+import { useState } from "react";
+import carouselSlide1 from "../assets/carouselSlide1.png";
+import carouselSlide3 from "../assets/carouselSlide3.png";
+import carouselSlide4 from "../assets/carouselSlide4.png";
+import carouselSlide5 from "../assets/carouselSlide5.png";
```

Comment:
@Fercas123 this file is missing so storybook cannot build, can you push and then we should be able to get this merged to labs,